mobile/lib/common/settings/emergency_kit_screen.dart

```diff
@@ -124,6 +128,64 @@ class _EmergencyKitScreenState extends State<EmergencyKitScreen> {
                           SizedBox(width: 10),
                           Text("Cleanup filling orders", style: TextStyle(fontSize: 16))
                         ])),
+                    const SizedBox(height: 30),
+                    Row(
```

Comment:
Why not having the delete button next to the channel?

Reply:
because I don't want somebody to accidentally delete their channel. IMO, it's safer to make that a bit more cumbersome :)